### ISSUE TITLE
Changing Document Root to Serve Laraval

### ISSUE THREAD

**Issue by dukesnuz** [susanBuck/dwa15-fall2017#58]
I am  using xampp
in httpd.conf 
If I use this:
```DocumentRoot "C:/xampp/htdocs/"```
```<Directory "C:/xampp/htdocs/">```
then in my browser I use:
```http://localhost/dwa/foobooks/public/```
then 
Laraval page opens

## The issue
If I use this:
```DocumentRoot "C:/xampp/htdocs/dwa/foobooks/public"```
```<Directory "C:/xampp/htdocs/dwa/foobooks/public">```
then in my browser I use:
```http://localhost/```
then I receive an error in browser
#### Error 403 displays
AND this does still work:
```http://localhost/dwa/foobooks/public/````

**Comment by susanBuck** [comment]
Can you confirm you stopped and restarted Apache via the XAMPP control panel after changing the document root to `C:/xampp/htdocs/dwa/foobooks/public` ? If in doubt, stop it and restart again.

Can you share a [gist](https://gist.github.com/) with the full contents of your XAMPP httpd.conf file?

**Comment by dukesnuz** [comment]
@susanBuck Here is the link to my [ httpd.conf'](https://gist.github.com/dukesnuz/dccbfb9a3a20913bc341b645ac38598d)

#### Note
I have been using MAMP mostly so far. Reason for now using xampp = I wanted to get familiar using xampp also. I did a reinstall of xampp also thinking it may solve the issue. It did not. I am wondering if has to do with permissions somewhere.

**Comment by dukesnuz** [comment]
I was not able to find the fix and rather than spend time on this, I decided to continue using mamp.  I did have xampp also working earlier in the class. Turns out mamp was not working either now. So I reinstalled mamp. All is fine with mamp now and localhost opens my Laravel app page.  Manmp seems so much more user friendly than xampp. Only issue with mamp is that I cannot create local domains. I can live with that.

 In the event I have to reinstall mamp or xampp again., I created a list of instructions we have used in the claass to set up a local server, 
[List of links to instructions](https://github.com/dukesnuz/setting-up-a-local-server/blob/master/steps)
